### ISSUE TITLE
The v2.0.0-alpha.2 APIs are not consistent with the ones as stated

### ISSUE THREAD

**Issue by cherry-geqi** [Automattic/node-canvas#971]
## Issue or Feature

I install the latest canvas@2.0.0-alpha.2 using `yarn add canvas@2.0.0-alpha.2` and no createCanvas nor loadImage found.

## Steps to Reproduce

```js
> require('canvas')
{ [Function: Canvas]
  _registerFont: [Function: _registerFont],
  backends:
   { ImageBackend: [Function: ImageBackend],
     PdfBackend: [Function: PdfBackend],
     SvgBackend: [Function: SvgBackend] },
  version: '2.0.0-alpha.2',
  cairoVersion: '1.14.10',
  jpegVersion: '8d',
  gifVersion: '4.2.3',
  freetypeVersion: '2.8.0',
  Context2d: { [Function: CanvasRenderingContext2D] parseFont: [Function] },
  PNGStream:
   { [Function: PNGStream]
     super_:
      { [Function: Readable]
        ReadableState: [Function: ReadableState],
        super_: [Object],
        _fromList: [Function: fromList] } },
  PDFStream:
   { [Function: PDFStream]
     super_:
      { [Function: Readable]
        ReadableState: [Function: ReadableState],
        super_: [Object],
        _fromList: [Function: fromList] } },
  JPEGStream:
   { [Function: JPEGStream]
     super_:
      { [Function: Readable]
        ReadableState: [Function: ReadableState],
        super_: [Object],
        _fromList: [Function: fromList] } },
  Image: { [Function: Image] MODE_IMAGE: 1, MODE_MIME: 2 },
  ImageData: [Function: ImageData],
  registerFont: [Function] }
>
```

## Your Environment
* Version of node-canvas (e.g. 1.4.0): v2.0.0-alpha.2
* Environment (e.g. node 4.2.0 on Mac OS X 10.8): node 6.9.1 on Mac OS X 10.12.6

**Comment by LinusU** [comment]
You are reading the documentation for the latest master commit, but that have yet to be included in a release. Please use the following link to read the documentation for version 2.0.0-alpha.2

https://github.com/Automattic/node-canvas/tree/v2.0.0-alpha.2

I wanted to give #943 a chance to land before cutting a new release, but when that one is ready we'll cut alpha.3

**Comment by LinusU** [comment]
Fixed in `v2.0.0-alpha.3` 🎉
